CustomerApp/src/main/java/FirstApp.java

```diff
@@ -0,0 +1,5 @@
+public class FirstApp {
+    public static void main(String[] args) {
```

Comment:
Package naming: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html.
My site is "anla.pp.ua". so, my company package will be "ua.pp.anla". Laboratory package is "laboratory" (for example). And the first module package is "restaurant". So full package of sources for first module will be in the next folder structure: src\main\java\ua\pp\anla\laboratory\restaurant.

AND.
Main POM.xml:
GroupID will be "ua.pp.anla".
ArtifactId will be "laboratory"

Restaurant app POM.xml:
GroupId = ua.pp.anla.laboratory
ArtifactId = restaurantapp

Customer app POM.xml:
GroupId = ua.pp.anla.laboratory
ArtifactId = customersapp

Note: for packages we use only low case symbols. Don't use "_", "-". "." and so on. the best package naming is one word.